scripts/check-api-compatibility.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Check API compatibility of all packages
+set -eu
+
+tmpdir=/tmp/compat-check
+
+package_name() {
+    node -pe "require('$1/package.json').name"
+}
+
+#----------------------------------------------------------------------
+
+echo "Listing packages..." >&2
+package_dirs=()
+package_names=()
+for dir in $(npx lerna ls -p); do
+    if [[ -f $dir/.jsii ]]; then
+        package_dirs+=("$dir")
+        package_names+=("$(package_name $dir)")
+    fi
+done
+
+#----------------------------------------------------------------------
+
+if ! ${SKIP_DOWNLOAD:-false}; then
+    echo "Filtering on existing packages on NPM..." >&2
+    for i in ${!package_dirs[@]}; do
+        echo -n "${package_names[$i]}... "
+        if npm view --loglevel silent ${package_names[$i]} > /dev/null; then
+            echo "Exists."
+        else
+            echo "NEW."
+            unset 'package_names[i]'
+            unset 'package_dirs[i]'
+        fi
+    done
+
+    rm -rf $tmpdir
+    mkdir -p $tmpdir
+
+    echo "Installing from NPM..." >&2
+    (cd $tmpdir && npm install --prefix $tmpdir ${package_names[@]})
```

Comment:
`--no-save` I guess

Reply:
In a tempdir? Don't care.

Reply:
Also, there is no `package.json` so where would it save :)